group-project/src/router.js

```diff
@@ -5,7 +5,7 @@ import Home from './Pages/Home.vue'
 import Achievement from './Pages/Achievement.vue'
 import Login from './Pages/Login.vue'
 import Adventure from './Pages/Adventure.vue'
-import MartinQuiz from './Pages/MartinQuiz.vue'
+import BergsQuiz from './Pages/MartinQuiz.vue'
```

Comment:
This is wrong I think. should the path be BergsQuiz?

Reply:
This is "correct" at the moment since the .vue file is still named MartinQuiz, i forgot to change the file name as well. I will change this next time :) Good spotting though haha!